webapp/src/ts/services/tasks-for-contact.service.ts

```diff
@@ -35,18 +35,19 @@ export class TasksForContactService {
         }
 
         // must be either a person type
-        if (type.person) {
+        if (type?.person) {
           return true;
         }
 
         // ... or a leaf place type
         return this.contactTypesService
           .getAll()
           .then(types => {
-            const hasChild = types.some(t => !t.person && t.parents && t.parents.includes(type.id));
+            const hasChild = types.some(t => !t.person && t.parents && t.parents.includes(type?.id));
```

Comment:
Maybe we can make this a little more efficient and check that `type` is actually defined and return false if it's not, before getting all contact types and such, I'd say that even before checking if the type is a person type.

Reply:
Good suggestion. Done.